frontend/src/index.tsx

```diff
@@ -35,16 +35,16 @@ createRoot(document.querySelector('#root') as HTMLElement).render(
                   </ProtectedRoute>
                 ),
               },
-              {
-                path: AppRoute.SIGN_IN,
-                element: <Auth />,
-              },
-              {
-                path: AppRoute.SIGN_UP,
-                element: <Auth />,
-              },
             ],
           },
+          {
+            path: AppRoute.SIGN_IN,
+            element: <Auth />,
+          },
+          {
+            path: AppRoute.SIGN_UP,
+            element: <Auth />,
+          },
```

Comment:
what is the purpose of this? if it's related to the issue of authenticated-user unnecessary call that you should return this back

Reply:
yeah, you are right. It is unnecessary now